lib/DDGC/DB/Result/GitHub/Comment.pm

```diff
@@ -0,0 +1,42 @@
+package DDGC::DB::Result::GitHub::Comment;
+# ABSTRACT:
+
+use Moose;
+use MooseX::NonMoose;
+extends 'DDGC::DB::Base::Result';
+use DBIx::Class::Candy;
+use namespace::autoclean;
+
+table 'github_comment';
+
+primary_column id          => {data_type => 'bigint', is_auto_increment => 1};
+unique_column  github_id   => {data_type => 'bigint', is_nullable => 0};
+column github_repo_id      => {data_type => 'bigint', is_nullable => 0};
+column github_user_id      => {data_type => 'bigint', is_nullable => 0};
+column number              => {data_type => 'bigint', is_nullable => 0};
+column body                => {data_type => 'text',   is_nullable => 0};
+column created_at          => {data_type => 'timestamp without time zone', is_nullable => 0};
+column updated_at          => {data_type => 'timestamp without time zone', is_nullable => 0};
+column gh_data             => {
+    data_type        => 'text',
+    is_nullable      => 0,
+    serializer_class => 'AnyJSON',
+    default_value    => '{}',
```

Comment:
Yeah, but since the field is not nullable, it should never get used.  Probably I don't need that line.

The AnyJSON serializer appears to be a custom serializer which is defined [here](https://github.com/duckduckgo/community-platform/blob/master/lib/DBIx/Class/InflateColumn/Serializer/AnyJSON.pm).  I think I should be using the standard JSON serializer.